presets/4.4/tune/barracuda_race_tune.txt

```diff
@@ -0,0 +1,68 @@
+#$ TITLE: Barracuda Racer Tune
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: 6S, race, racing, beginner, 5 inch, 5", viki, 2024
+#$ AUTHOR: Viki Baarathi
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: This tune is ideal for beginners who are taking up drone racing. It's light on the motors yet allows for good performance.
+
+#$ DESCRIPTION: ### Requirements:
+#$ DESCRIPTION: - 5 inch race quads with 6S packs.
+#$ DESCRIPTION: - Works with Analog or HDZero
+#$ DESCRIPTION: - DSHOT600 with bidirectional DSHOT enabled. This is very important.
+#$ DESCRIPTION: - Stack nuts must be present and tight. The ones below the FC. Some call them the golden nuts or nuts of glory. 
+#$ DESCRIPTION: - Motor screws should be tight and arms should be solid and not gummy. 
+#$ DESCRIPTION: - Special thanks to testers Meor, Imad, Muqrie, AdilzFPV
+#$ DESCRIPTION: ### Recommendation:
+#$ DESCRIPTION: - Apply Barracuda filters
+#$ DESCRIPTION: - Apply RClink of ExpressLRS 250Hz or ExpressLRS 500Hz for racing
+#$ DESCRIPTION: ### BLHeli32 setting
+#$ DESCRIPTION: - Set low and high frequency to 48
+#$ DESCRIPTION: - Rampup power 25%
+#$ DESCRIPTION: - Motor timing 16 deg
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/442
+#$ INCLUDE_WARNING: misc/warnings/en/dshot.txt
+
+#$ OPTION BEGIN (CHECKED): Recommended Barracuda Filters
+    #$ INCLUDE: presets/4.3/filters/defaults.txt
+    set simplified_gyro_filter_multiplier = 140
+    set gyro_lpf2_static_hz = 0
+
+#$ OPTION END
+
+#$ INCLUDE: presets/4.4/tune/defaults.txt
+set pidsum_limit = 1000
+set pidsum_limit_yaw = 1000
+set simplified_pids_mode = RPY
+set simplified_master_multiplier = 60
+set simplified_i_gain = 110
+set simplified_dmax_gain = 0
+set simplified_feedforward_gain = 80
+
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set dshot_bidir = ON
+set motor_pwm_protocol = DSHOT600
+
+simplified_tuning apply
+
+#$ OPTION_GROUP BEGIN: Recommended Race RC Links
+
+    #$ OPTION BEGIN (UNCHECKED): ELRS 250Hz Race Setup
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+        #$ INCLUDE: presets/4.3/rc_link/generic/250hz_race.txt
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): ELRS 500Hz Race setup
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+        #$ INCLUDE: presets/4.3/rc_link/generic/250hz_race.txt
```

Comment:
```suggestion
        #$ INCLUDE: presets/4.3/rc_link/generic/500hz_race.txt
```

Reply:
please also confirm you want `_race`. The freestyle option might feel less twitchy for beginners, but it's totally up to your preferences.

Reply:
Yes. I've tested with a large number of pilots and feedback is amazing. This setup works well.